addons/l10n_ec/data/account_tax_template_vat_data.xml

```diff
@@ -0,0 +1,785 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data >
+        <!-- 
+        VAT TAXES *IMPUESTO AL VALOR AGREGADO
+        -->
+        <record id="tax_vat_411" model="account.tax.template">
+        	<!-- IVA EN VENTAS LOCALES (EXCLUYE ACTIVOS FIJOS) -->
+            <field name="name">IVA 12%</field>
+            <field name="type_tax_use">sale</field>
+            <field name="amount_type">percent</field>
+            <field name="sequence">9</field>
+            <field name="amount">12.0</field>
+            <field name="description">IVA 12%</field>
+            
+            
+            <field name="chart_template_id" ref="l10n_ec.l10n_ec_ifrs"/>
+            <field name="tax_group_id" ref="ec_tax_group_vat12"/>
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0,0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'tag_ids': [ref('tax_tag_411')],
```

Comment:
@Stentor instead of this we shoudl use + or - tag_ids.

Reply:
This will take some time, i need to check with a tax expert which are which.